.gitignore

```diff
@@ -24,3 +24,34 @@ DerivedData
 *.xcuserstate
 # Jetbrains
 .idea
+
+# Gradle
+/build/
+/RNTester/android/app/build/
+/RNTester/android/app/gradle/
+/RNTester/android/app/gradlew
+/RNTester/android/app/gradlew.bat
+/ReactAndroid/build/
+
+# Buck
+.buckd
+buck-out
+/ReactAndroid/src/main/jni/prebuilt/lib/armeabi-v7a/
+/ReactAndroid/src/main/jni/prebuilt/lib/x86/
+/ReactAndroid/src/main/gen
+
+# Watchman
+.watchmanconfig
+
+# Android
+.idea
+.gradle
+local.properties
+*.iml
+/android/
```

Comment:
I am wondering if this has broken android? I don't get android folder in my node_modules/react-native-sound after `npm install react-native-sound --save` hence the `react-native link react-native-sound` fails to link android and only links iOS and consequently getting RNSound.IsAndroid undefined.

Reply:
YIKES!!
Not sure how that one android folder made it in. I'm going to revert the change.